core/src/cluster_info_vote_listener.rs

```diff
@@ -409,17 +426,23 @@ impl ClusterInfoVoteListener {
             if time_since_lock.elapsed().as_millis() > GOSSIP_SLEEP_MILLIS as u128 {
                 let bank = poh_recorder.lock().unwrap().bank();
                 if let Some(bank) = bank {
-                    let last_version = bank.last_vote_sync.load(Ordering::Relaxed);
-                    let (new_version, msgs) = verified_vote_packets.get_latest_votes(last_version);
-                    inc_new_counter_info!("bank_send_loop_batch_size", msgs.packets.len());
-                    inc_new_counter_info!("bank_send_loop_num_batches", 1);
```

Comment:
will we need these two counters, or their equivalents?

Reply:
good catch, cleaned up and added those here https://github.com/solana-labs/solana/pull/20873/commits/95a30fbab174d1ddce399d6093fdb8b28435ed63